src/common/options.cc

```diff
@@ -4892,9 +4896,10 @@ std::vector<Option> get_global_options() {
     Option("bluestore_no_per_pool_stats_tolerance", Option::TYPE_STR, Option::LEVEL_ADVANCED)
     .set_default("until_repair")
     .set_flag(Option::FLAG_RUNTIME)
-    .set_enum_allowed({"enforce", "until_repair", "until_fsck"})
+    .set_enum_allowed({"enforce", "until_repair", "until_fsck", "fix_on_mount"})
     .set_description("Specified how to treat lack of per-pool stats, e.g. caused by an upgrade")
     .set_long_description(
+      "'fix_on_mount' will automativally fix the issue when detected during store mounting, "
```

Comment:
I still think we should simplify this option.. see my ``bluestore fsck behavior with legacy stats etc`` thread on the dev list.  If we do that, the fix_on_mount would just become a new bool option.